cmd/kops/create_cluster.go

```diff
@@ -486,16 +486,8 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 			bastionGroup.ObjectMeta.Name = "bastions"
 			instanceGroups = append(instanceGroups, bastionGroup)
 
-			// Logic to handle default bastion names
-			if c.DNSZone != "" {
```

Comment:
I don't understand what it was doing in retrospect.  I commented here: https://github.com/kubernetes/kops/issues/1322#issuecomment-271294740